.gitignore

```diff
@@ -1,4 +1,11 @@
 *.pyc
+*.tar.gz
+*.tar.gz*
 
 plots/
 temp/
+
+data/
+!data/CustomPhotonField/
+!data/dint/
```

Comment:
DINT and EleCa are removed from CRPropa. Also remove it here.

Reply:
Done. Also removed the dint and eleca data directories.